Sources/BraintreePayPal/BTPayPalCheckoutRequest.m

```diff
@@ -0,0 +1,55 @@
+#import "BTPayPalCheckoutRequest_Internal.h"
+#import "BTPayPalRequest_Internal.h"
+
+@implementation BTPayPalCheckoutRequest
+
+- (instancetype)initWithAmount:(NSString *)amount {
```

Comment:
This may be outside the scope of this PR, but what do you think about removing `amount` from the constructor and setting it in the same way as the other parameters?